src/katgpucbf/utils.py

```diff
@@ -105,14 +106,19 @@ def callback(phase: str, info: dict) -> None:
     gc.callbacks.append(callback)
 
 
-def parse_dither(value: str) -> DitherType:
-    """Parse a string into a dither type."""
-    # Note: this allows only the non-aliases, so excludes DEFAULT
-    table = {member.name.lower(): member for member in DitherType}
+def parse_enum(name: str, value: str, cls: type[_E]) -> _E:
+    """Parse a command-line argument into an enum type."""
+    table = {member.name.lower(): member for member in cls}
     try:
         return table[value]
     except KeyError:
-        raise ValueError(f"Invalid dither value {value} (valid values are {list(table.keys())})") from None
+        raise ValueError(f"Invalid {name} value {value} (valid values are {list(table.keys())})") from None
+
+
+def parse_dither(value: str) -> DitherType:
+    """Parse a string into a dither type."""
+    # Note: this allows only the non-aliases, so excludes DEFAULT
+    return parse_enum("dither", value, DitherType)
```

Comment:
Must admit, I'm not seeing how this `Note:` is enforced. Perhaps that's a conversation for another time.